src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -255,11 +256,18 @@ privilege : CREATE
           | MODULE_READ
           | MODULE_WRITE
           | WEBSOCKET
+          | EDGE_TYPES edgeTypes = edgeTypeList
           | LABELS labels=labelList
           ;
 
 privilegeList : privilege ( ',' privilege )* ;
 
+edgeTypeList : '*' | listOfEdgeTypes ;
```

Comment:
do we have an ASTERISK defined already?

Reply:
Good catch. We missed this altogether. We can leave iit as-is for this PR and create additional one to fix this in label-list also OR do the fix for labels as well in this PR

Reply:
I dont understand what is the problem here ?